### PR TITLE
Association CIQUAL AGB (starting from line 997)

### PR DESCRIPTION
**Pull request by bernardlemoullec**
Previous one was https://github.com/openfoodfacts/openfoodfacts-server/pull/3110

### CODE REVIEW

**Comment by stephanegigandet** [comment]
I'm going to build the taxonomy manually so that we can get the results of the tests.

**Comment by stephanegigandet** [comment]
./build_tags_taxonomy.pl categories publish

gives this:

[Fri Mar 27 09:52:48 2020] build_tags_taxonomy.pl: Empty entry at line 34664 in /srv/off/taxonomies/categories.txt

<en:Sauces
fr:Sauces bourguignonnes, Sauce bourguignonne préemballée
en:Prepacked Burgundy-style sauce, Prepackaged Burgundy-style sauce
fr:
ciqual_food_code:en:11120
ciqual_food_name:en:Burgundy-style sauce, prepacked
ciqual_food_name:fr:Sauce bourguignonne, préemballée

I'll fix it.

**Comment by stephanegigandet** [comment]
More errors:

Errors in the categories taxonomy definition:
ERROR - en:fat-free-plain-fresh-cream-cheese already is a synonym of fat-free-plain-fresh-cream-cheese (en:fat-free-plain-fresh-cream-cheese) - cannot make a synonym of fat-free-plain-petit-suisse for en:fat-free-plain-petit-suisse
ERROR - fr:glaces-au-yaourt already is a synonym of glaces-au-yaourt (fr:glaces-au-yaourt) - cannot make a synonym of yaourts-glaces for en:frozen-yogurts
ERROR - fr:saucisse-aux-lentilles already is a synonym of saucisses-aux-lentilles () - cannot make a synonym of petits-sales-aux-lentilles for fr:petits-sales-aux-lentilles
ERROR - fr:tarte-au-saumon already is a synonym of tarte-au-saumon (en:salmon-tart) - cannot make a synonym of quiches-au-saumon for en:quiches-with-fish
ERROR - fr:fromage-de-tete already is a synonym of fromage-de-tete (fr:fromage-de-tete) - cannot make a synonym of pates-de-tete for en:head-cheese
ERROR - en:twelfth-night-cake already is a synonym of king-cakes (en:twelfth-night-cake) - cannot make a synonym of twelfth-night-cake for en:twelfth-night-cake
ERROR - fr:chou-a-la-creme-patissiere already is a synonym of profiteroles (en:chou-pastry-filled-with-custard) - cannot make a synonym of chou-a-la-creme-patissiere for en:chou-pastry-filled-with-custard
ERROR - en:oriental-pastry-with-almonds-and-syrup already is a synonym of baklava () - cannot make a synonym of gazelle-horn for en:gazelle-horn
ERROR - fr:patisserie-orientale-aux-amandes-et-sirop already is a synonym of baklava () - cannot make a synonym of corne-de-gazelle for en:gazelle-horn
ERROR - fr:hareng-marine already is a synonym of harengs-marines (en:marinated-atlantic-herring) - cannot make a synonym of hareng-marine for en:marinated-atlantic-herring
ERROR - fr:graines-de-lin already is a synonym of lins (en:flaxseed) - cannot make a synonym of graines-de-lin for en:flaxseed
ERROR - fr:barre-de-cereales-aux-amandes already is a synonym of barres-de-cereales-aux-amandes () - cannot make a synonym of barre-cerealiere-aux-amandes for en:cereal-bar-with-almonds
ERROR - fr:barre-de-cereales-aux-noisettes already is a synonym of barres-de-cereales-aux-noisettes () - cannot make a synonym of barre-cerealiere-aux-noisettes for en:cereal-bar-with-hazelnuts
<h1>Software error:</h1>
<pre>Errors in the categories taxonomy definition at ProductOpener/Tags.pm line 981.
</pre>
<p>
For help, please send mail to this site's webmaster, giving this error message
and the time and date of the error.

</p>
[Fri Mar 27 09:58:02 2020] build_tags_taxonomy.pl: Errors in the categories taxonomy definition at ProductOpener/Tags.pm line 981.

**Comment by bernardlemoullec** [comment]
Hello Stéphane, I'm not sure how I am supposed to interpret the errors in the categories taxonomy definition... Sorry!

File: taxonomies/categories.result.txt [hunk 1]
```diff
@@ -863,6 +920,14 @@ fi:purkitetut artisokat
 fr:Artichauts en conserve
 nl:Artisjokken en blik
 
+< en:Artichokes
+< en:Frozen foods
+en:Frozen row artichoke bottom
```
Comment (alex-marty) [inline taxonomies/categories.result.txt]:
```suggestion
en:Frozen raw artichoke bottom
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -43344,12 +46599,27 @@ fi:artisokan sydämet
...
+
 <en:Artichokes
 <en:Fresh vegetables
 en:Fresh artichokes
 fr:Artichauts frais
 season_in_country_fr:en: 5,6,7,8,9
 
+<en:Artichokes
+<en:Frozen foods
+en:Frozen row artichoke bottom
```
Comment (alex-marty) [inline taxonomies/categories.txt]:
```suggestion
en:Frozen raw artichoke bottom
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -34663,6 +36707,34 @@ nl:Kalkoenenvleesmaaltijden
...
+
+<en:Turkey dishes
+en:Cooked turkey liver
+fr:Foie de dinde cuit
+ciqual_food_code:en:40118
+ciqual_food_name:en:Liver, turkey, cooked
+ciqual_food_name:fr:Foie, dinde, cuit
+
+<en:Turkey dishes
+en:Raw liver turkey
```
Comment (teolemon) [inline taxonomies/categories.txt]:
```suggestion
en:Raw turkey liver
```

File: taxonomies/categories.txt (hunk 2 above)
Comment (teolemon) [inline taxonomies/categories.txt]:
```suggestion
en:Frozen raw artichoke bottom
```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -48554,22 +52059,125 @@ nl_be:Taartdeeg
...
+ciqual_food_code:en:23414
+ciqual_food_name:en:Short crust pastry, pure butter, raw
+ciqual_food_name:fr:Pâte brisée, pur beurre, crue
 
 <en:Pie dough
-en:Puff pastry sheets
+en:Puff pastry sheets, Cooked puff pastry
 de:Blätterteig Platten
-fr:Pâtes feuilletées, Pâte feuilletée, Pâtes à tarte feuilletées
+fr:Pâtes feuilletées, Pâte feuilletée, Pâtes à tarte feuilletées, Pâte feuilletée cuit
```
Comment (teolemon) [inline taxonomies/categories.txt]:
```suggestion
fr:Pâtes feuilletées, Pâte feuilletée, Pâtes à tarte feuilletées, Pâte feuilletée cuite
```

**Comment by teolemon** [comment]
Merged, but we need to fix those 3: 
fr:Pâtes feuilletées, Pâte feuilletée, Pâtes à tarte feuilletées, Pâte feuilletée cuite
en:Frozen raw artichoke bottom
en:Raw turkey liver

**Comment by bernardlemoullec** [comment]
Roger that, thanks

**Pull request merged**